### ISSUE TITLE
Bump Svelte to 4?

### ISSUE THREAD

**Issue by paolodina** [bryanmylee/svelte-keyed#9]
```bash
$ pnpm i
Packages: +26 -14
++++++++++++++++++++++++++--------------
Progress: resolved 291, reused 269, downloaded 0, added 0, done

devDependencies:
- svelte 3.59.2
+ svelte 4.0.5

 WARN  Issues with peer dependencies found
.
└─┬ svelte-headless-table 0.17.4
  └─┬ svelte-keyed 1.1.6
    └── ✕ unmet peer svelte@^3.49.0: found 4.0.5

Done in 2.4s
```

**Comment by bryanmylee** [comment]
Yep! Will bump.

Crazy how nothing really changed with Svelte 4 yet it still created a bunch of busy work for devs everywhere.

The webdev scene is ridiculous.

**Comment by bryanmylee** [comment]
Closed by #8
